app/classifier/index.cjsx

```diff
@@ -216,26 +232,26 @@ Classifier = React.createClass
             </p>
 
             {if @props.demoMode
-              <p style={{textAlign: 'center'}}>
-                <i className="fa fa-trash"></i>{' '}
+              <p style={{ textAlign: 'center' }}>
+                <i className="fa fa-trash" />{' '}
                 <small>
                   <strong>Demo mode:</strong>
                   <br />
                   No classifications are being recorded.{' '}
-                  <button type="button" className="secret-button" onClick={@onChangeDemoMode}>
+                  <button type="button" className="secret-button" onClick={@props.onChangeDemoMode.bind(this, false)}>
```

Comment:
This is throwing this React warning: `bind(): React component methods may only be bound to the component instance.` which is a new one to me, so I'm not sure what issues it might cause.